main.py

```diff
@@ -37,174 +21,53 @@ class ItemsDB():
-        data = Dict
-
-        try:  # todo add 4xx 5xx handler and retry
-            headers = {
-                "Content-Encoding": "zstd",  # Compression format
-                "Content-Type": "application/json; charset=utf-8"  # JSON format with UTF-8 character set
-            }
-            response = requests.get(url, timeout=30, headers=headers)  # Set a timeout of 3 seconds
-            data = response.json()
-        except requests.exceptions.Timeout:
-            print("Request timed out!")
-
-        self.storage.data.update(self.parce_info(data))
-        return data["totalCount"]
-
-    def parce_info(self, raw_date: Dict) -> Dict:
-        real_state = {}
-        for i in raw_date["items"]:
-            real_state[i["id"]] = {
-                "coords": Coords(i["coords"]["lat"], i["coords"]["lng"]),
-                "price": int(re.sub(r"\D", "", i["normalizedPrice"]))
-            }
-        return real_state
-
-
-class ViewData:
-
-    def __init__(self, storage: Storage):
-        self.storage = storage
-
-    def create_map(self, x: float, y: float):
-        city_center = [x, y]
-        data_points = []
-        print("create map")
-        for val in self.storage.return_with_filter(x + 0.02949858216,
-                                                   x - 0.02949858216,
-                                                   y + 0.05012512207,
-                                                   y - 0.05012512207).values():
-            data_points.append([val["coords"].x, val["coords"].y, val["price"]])
-
-        # Создаём базовую карту
-        m = folium.Map(location=city_center, zoom_start=12)
-
-        # Добавляем тепловую карту
-        heat_data = [[point[0], point[1], point[2]] for point in data_points]  # Координаты и интенсивность
-        HeatMap(heat_data).add_to(m)
-        m.save("heatmap.html")
-        print("map created")
 
 # storage = Storage()
 # coords_queue = RequestedCoords()
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
 
 
-
-
-def http_server(storage: Storage, coords_queue: RequestedCoords):
-
+def http_server(storage: Storage, coords_queue: RequestedCoords) -> FastAPI:
     app = FastAPI()
+
     @app.get("/get/")
-    async def get_file(x: float, y: float):
-        print("start request page")
-        coords_queue.coords.append(Coords(x,y))
+    async def get_file(x: float, y: float) -> FastAPI:
+        logger.info("Start request page")
+        coords_queue.coords.append(Coords(x, y))
         view_data = ViewData(storage)
         view_data.create_map(x, y)
-        print("page generated")
-        file_path = f"./heatmap.html"
-        print("handle request")
+        logger.info("Page generated")
+        file_path = "./heatmap.html"
+        logger.info("Handle request")
         if os.path.exists(file_path):
             return FileResponse(file_path)
         else:
             raise HTTPException(status_code=404, detail="File not found")
```

Comment:
usually we dont have logic in the routers, rather they utilize functions and methods to process the request. And this part deserved to be in another function to process file logic